package.json

```diff
@@ -71,7 +72,11 @@
     "eslint-plugin-standard": "^4.0.0",
     "html-loader": "0.5.5",
     "html-webpack-plugin": "3.2.0",
+    "mini-css-extract-plugin": "^0.5.0",
+    "node-sass": "^4.11.0",
+    "normalize-scss": "^7.0.1",
```

Comment:
Let's not start adding unnecessary packages, we can normalize css by including a CDN embedded on the page.

Reply:
With this comment I'm referring to all 5 of the packages just added.

Reply:
What are you talking about?

Reply:
Why would we include an external dependency to test the project locally?

Reply:
I'm saying we don't need sass, so let's not add it